gradle.properties

```diff
@@ -11,7 +11,7 @@ forge_version=40.1.0
 //forge_ats_enabled=false
 
 # Fabric
-fabric_version=0.51.0+1.18.2
+fabric_version=0.51.1+1.18.2
```

Comment:
Might want to revert this change; no reason to break backwards compat if I don't need to